01-js-intro-and-jquery/07/js/scripts.js

```diff
@@ -0,0 +1,30 @@
+$(document).ready(function() {
+	var alias = '';
+	var focusAlias = function() {	
+		$('.alias').focus(); 
+	};
+	var getResponse = function() {
+		alias = $('.alias').val();
+
+		$.ajax({
+   url: "http://bootcamp.aws.af.cm/welcome/" + alias,
+   type: 'GET',
+   dataType: "json",
+   success: function( data ) {
+   	if(data.error) {
```

Comment:
This will only work if our response returns an ``error`` object.
You could try using ``error`` callback provided by ``$.ajax``.